client/android/build.gradle

```diff
@@ -12,6 +12,7 @@ buildscript {
        threetenabpVersion = '1.1.1'
        groupName = 'org.amnezia.vpn'
        minSdkVer = '24'
+       cmakeMinVersion = "3.25.0+"
```

Comment:
@kolobchanin are you sure about minimum supported CMake version? 

I tested it with 3.24.2 and it works.

Reply:
We moved whole application to cmake 3.25+, not only this part.